vllm/config.py

```diff
@@ -2687,7 +2687,7 @@ class CompilationConfig(BaseModel):
             outside of compilation.
             TODO: move outside cudagraph logic into compilation.
             torch.compile will handle cudagraph capture logic in the future.
-        - cudagraph_capture_sizes: sizes to capture cudagraph.
+        - capture_sizes: sizes to capture cudagraph.
```

Comment:
Why change the name of this field if it is still only used for cudagraphs?

Reply:
unified in [29b08b7](https://github.com/vllm-project/vllm/pull/12243/commits/29b08b7838588755b566486ab3a67a2bae23f65c)